ui/tests/helpers/noop-all-api-requests.js

```diff
@@ -0,0 +1,20 @@
+import Pretender from 'pretender';
+
+const noop = response => {
+  return function() {
+    return [response, { 'Content-Type': 'application/json' }, JSON.stringify({})];
+  };
+};
+
+export default function(options = { usePassthrough: false }) {
+  return new Pretender(function() {
+    let fn = noop();
+    if (options.usePassthrough) {
+      fn = this.passthrough;
+    }
+    this.post('/v1/**', fn);
```

Comment:
seems like it'd be better to take path as an arg here rather than hardcode it but for your use-cases below i suppose it makes sense.

Reply:
Yeah, right now we don't make any requests outside of that namespace so felt better to just hardcode it for now so.